components/SlidingRowView/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import RowView, { RowViewProps } from "../RowView";
+import { Animated, Easing, StyleProp, ViewStyle } from "react-native";
+
+interface Props extends RowViewProps {
+  index: number;
+  style?: StyleProp<ViewStyle>;
+}
+
+const initialPosition = -200;
+
+export default class SlidingRowView extends React.PureComponent<Props> {  
+  position = new Animated.Value(initialPosition);
+  opacity = new Animated.Value(0.5);
+
+  componentDidMount() {
+    this.animateAddOperation();
+  }
+
+  animateAddOperation() {
+    Animated.parallel([
+      Animated.timing(this.position, {
+        toValue: 0,
+        duration: 50,
+        easing: Easing.ease
+      }),
+      Animated.timing(this.opacity, {
```

Comment:
Intentionally changing the opacity with `parallel`. We can show you later in your animation to present the cells from top to bottom how to do it with `interpolate`. Thoughts?

Reply:
sounds good.